substrate-archive/src/actors.rs

```diff
@@ -157,15 +144,79 @@ where
 	}
 }
 
-struct Actors<B: BlockT + Unpin, D: ReadOnlyDb + 'static>
+struct Actors<Block: Send + Sync + 'static, H: Send + Sync + 'static, Db: Send + Sync + 'static> {
```

Comment:
are these trait bounds needed?

Reply:
Yeah, `Address` requires Sync + Send in `xtra`